packages/graphql/src/modules/user/domain/User.ts

```diff
@@ -1,49 +1,95 @@
-export type Providers = 'facebook' | 'google' | 'apple';
+namespace User {
```

Comment:
How about just export a user constant instead use namespace? I think this kind of syntax is outdated.